ontotext-yasgui-web-component/src/services/utils/time-utils.ts

```diff
@@ -0,0 +1,9 @@
+export class TimeUtils {
```

Comment:
Change to just FunctionUtils or maybe just Debounce

Reply:
Renamed to "Debounce"